README.md

```diff
@@ -269,6 +269,32 @@ module.exports = {
 };
 ```
 
+### `postTransformPublicPath`
+
+Type: `Function`
+Default: `undefined`
+
+Specifies a custom function to post-process the generated public path. This can be used to prepend or append dynamic global variables that are only available at runtime, like `__webpack_public_path__`. This would not be possible with just `publicPath`, since it stringifies the values.
+
+**webpack.config.js**
+
+```js
+module.exports = {
+  module: {
+    rules: [
+      {
+        test: /\.(png|jpg|gif)$/,
+        loader: 'file-loader',
+        options: {
+          publicPath: '/some/path/',
+          postTransformPublicPath: (p) => `__webpack_public_path__ + ${p}`,
+        },
+      },
+    ],
+  },
+};
+```
+
```

Comment:
Oh, i forgot, let's add Example use case for this

Reply:
i pushed an example, is this what you mean?

Reply:
oh, i see, just think about additional section in `Example` in README

Reply:
ups, i forgot to push the commit i was talking about 😅
i meant this one: https://github.com/webpack-contrib/file-loader/pull/334/commits/460127db553624d14b6af3664986fd41d02082e0